src/pybind/mgr/cephadm/tests/test_service_discovery.py

```diff
@@ -102,7 +105,7 @@ def test_get_sd_config_prometheus(self):
             assert 'targets' in entry
 
         # check content
-        assert cfg[0]['targets'] == ['node0:9283', 'node1:9283']
+        assert cfg[0]['targets'] == ['node0:9283']
```

Comment:
Why the second target has dissapear?

Reply:
> Why the second target has dissapear?

Very good observation! in fact in the new version adding TLS to the standby is not straightforward (we have to setup all the certificates etc) so I limited the reported targets to the active mgr only. Standby was returning an empty value for the target `http://stdby-mgr-ip:port`.